### PR TITLE
Repo sync for protected CLA branch

### PR DESCRIPTION
**Pull request by opbld15**
The pull request is created from public637915981872726566 to public to fix git push error for protected CLA branch

### CODE REVIEW

**Comment by officedocspr4** [comment]
## STOP  Caution: Large pull request

This **public** repo applies limits on the number of files that can be merged in a single pull request (PR):

- **Warning** Warning message and ability to unblock PR using the *Merge large pull request*<sup>\*</sup> label - 30
- **Block** Block message and PR is blocked - 100

This pull request will change **69** files, which exceeds the **warning** limit of **69**.

Before this pull request can be merged, you must review the list of files it contains and their changes to confirm they are correct and intentional. **This includes files you did not change and/or do not own**. If there are changes you aren't familiar with, contact the individuals who submitted the commits included in this pull request and review the changes with them.

**The individual who merges this pull request is responsible for ensuring the changes included in it are correct, and for resolving any issues that might result in merging it.**

### To merge this pull request

If you have determined that the changes in this pull request are correct, add the **Merge large pull request**<sup>\*</sup> label to it, wait for the **max/pr-file-count** check to pass, and then click **Merge**. To add a label, click the gear icon next to **Labels** and then select **Merge large pull request**<sup>\*</sup>.

Label UI screenshot

### To reject this pull request

If you can't confirm the changes in this pull request are correct, click **Close pull request** at the bottom of this page to close it without merging. Work with your team to revert any changes that aren't correct.

<sup>\*</sup> You must have write or triage access to this repo to add labels.

File: microsoft-365/business-premium/m365bp-device-groups-mdb.md
Comment (JohanFreelancer9) [inline microsoft-365/business-premium/m365bp-device-groups-mdb.md:84]:
```suggestion
You can also enter the [Azure AD portal](https://portal.azure.com) from the Microsoft 365 admin center. In the [Microsoft 365 admin center](https://admin.microsoft.com/), choose **All admin centers**, and then choose **Azure Active Directory**.
```

File: microsoft-365/business-premium/m365bp-device-groups-mdb.md
Comment (JohanFreelancer9) [inline microsoft-365/business-premium/m365bp-device-groups-mdb.md:98]:
```suggestion
After choosing a category, the device is automatically added to the corresponding group. If a device is already enrolled before you configure categories, the user sees a notification about the device on the company portal website. This lets the user know to select a category the next time they access the company portal app on iOS/iPadOS or Android.
```

File: microsoft-365/business-premium/m365bp-review-threats-take-action.md
Comment (JohanFreelancer9) [inline microsoft-365/business-premium/m365bp-review-threats-take-action.md:55]:
```suggestion
1. Select a threat to see more details about the threat.
```

File: microsoft-365/business-premium/m365bp-review-threats-take-action.md
Comment (JohanFreelancer9) [inline microsoft-365/business-premium/m365bp-review-threats-take-action.md:61]:
```suggestion
You can use Microsoft Endpoint Manager to manage threat detections as well. First, all devices whether Windows, iOS, or Android, must be [enrolled in Intune](/mem/intune/enrollment/windows-enrollment-methods) (part of Microsoft Endpoint Manager).
```

File: microsoft-365/business-premium/m365bp-validate-settings-on-android-or-ios.md
Comment (JohanFreelancer9) [inline microsoft-365/business-premium/m365bp-validate-settings-on-android-or-ios.md:31]:
```suggestion
```

File: microsoft-365/compliance/retention-cmdlets.md
Comment (JohanFreelancer9) [inline microsoft-365/compliance/retention-cmdlets.md:45]: Please look at the rich diff (preview) - the following two tables do not render correctly. Kindly correct the formatting.

File: microsoft-365/compliance/retention-cmdlets.md
Comment (JohanFreelancer9) [inline microsoft-365/compliance/retention-cmdlets.md:62]:
```suggestion
```

File: microsoft-365/compliance/retention.md
Comment (JohanFreelancer9) [inline microsoft-365/compliance/retention.md:549]:
```suggestion
Use [Office 365 Security & Compliance Center PowerShell](/powershell/exchange/scc-powershell) for Purview retention cmdlets that support configuration at-scale, scripting for automation, or that might be necessary for advanced configuration scenarios.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @opbld15, kindly commit the suggested changes and correct the table formatting as indicated. Many thanks.

**Pull request closed without merging**
